company/strategy.md

```diff
@@ -1,36 +1,90 @@
-# Sourcegraph strategy
+# Sourcegraph master plan
+
+Everything we do stems from our [purpose](#purpose), to make it so **everyone can code**. We'll accomplish that over the next 30 years by following our strategy:
+
+- 30 years
+  - [Purpose](#purpose)
+  - [Mission](#mission)
+  - [Values](#values)
+  - [Big Code](#big-code)
+- 3 years
+  - [Principles](#principles)
+  - [Assumptions](#assumptions)
+- 1 year
+  - [1-year vision](#1-year-vision)
+  - [Product direction](#product-direction)
+  - [OKRs](#okrs)
 
 ## Purpose
 
-Sourcegraph exists to create a world where everyone builds software. A world where everyone, not just ~0.2% of the world's population (people who can code today), builds software will see faster and more broadly beneficial technological progress.
+We want to make it so **everyone can code**. A world where everyone, not just ~0.1% of the world population, can code will see faster and more broadly beneficial technological progress.
 
-Everyone, in every community, in every country, and in every industry--not just the ones working at the half-dozen dominant tech companies--needs to be able to build software with the very best tools, libraries, and technologies. This is the only way the world will sustain broad economic growth and build the innovations we need in transportation, agriculture, communication, health care, energy, construction, etc.
+### Background
 
-> If you think everyone building software will never happen, ask yourself: is coding more like reading and writing (which virtually everyone does) or publishing books (which ~0.1% of the population does)?
+For thousands of years after writing was invented, most people remained illiterate. Universal literacy seemed unlikely. Is it really possible that every human would be capable of reading and writing? How would literacy benefit the average person? As we now know, every human is capable of and benefits immensely from literacy.
+
+More recently, around 1976, just 0.2% of the world's population used computers. Two tiny companies sought to make computing universal: Apple's vision then was to create a "bicycle for the mind" in the form of a computer, and Microsoft wanted to put a computer "on every desk and in every home". Though it seemed unlikely at the time, as we now know, everyone is capable of and benefits immensely from having a computer (or a phone).
+
+Today, only about 0.1% of the world's population can code. That tiny group has built software that runs the modern world and improves the lives of billions of people. Think of the possibilities if everyone was able to code. All around the world, more people would be able to solve problems and improve their lives by building software. We don't know exactly what these billions of coders will create, but we know that this will bring faster and more broadly beneficial technological progress.
```

Comment:
```suggestion
Today, only about 0.1% of the world's population can code. That tiny group has built software that runs the modern world and improves the lives of billions of people. Think of the possibilities if everyone could code. All around the world, more people would be able to solve problems and improve their lives by building software. We don't know exactly what these billions of coders will create, but we know that this will bring faster and more broadly beneficial technological progress.
```